conans/test/integration/command_v2/search_test.py

```diff
@@ -0,0 +1,221 @@
+import textwrap
+from collections import OrderedDict
+
+import pytest
+
+from conans.test.utils.tools import TestClient, TestServer
+
+
+class TestSearch:
+
+    @pytest.fixture(autouse=True)
```

Comment:
If the fixture is not used in 50% of the cases, better inject it explicitly

Reply:
Fixed